packages/backend/src/open_payments/quote/service.ts

```diff
@@ -59,7 +57,9 @@ async function getQuote(
   deps: ServiceDependencies,
   options: GetOptions
 ): Promise<Quote | undefined> {
-  return Quote.query(deps.knex).get(options).withGraphFetched('asset')
+  return Quote.query(deps.knex)
+    .get(options)
+    .withGraphFetched('[asset, fee.asset]')
```

Comment:
Just fee should work. Originally I was thinking I'd use the asset associated with the fee but I did not (and I suppose we could just use the quote one since we know it's the same).